src/Ago.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import moment from 'moment';
+
+import TimeAgo from 'javascript-time-ago';
+import localeEN from 'javascript-time-ago/locale/en';
+
+// Setup english
+TimeAgo.locale(localeEN);
+
+/**
+ * Fuzzy Date Component
+ */
+export default class Ago extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.convertTime = new TimeAgo(props.locale);
+  }
+
+  componentDidMount() {
+    if (moment(this.props.date).isSame(moment(), 'day')) {
+      this.intervalId = setInterval(() => {
+        this.forceUpdate();
+      }, this.props.timeout);
+    }
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.intervalId);
+  }
+
+  render() {
+    const time = moment(this.props.date).add(this.props.offset, 'ms');
+    if (!time.isValid()) {
+      return null;
+    }
+
+    // since Twitter style TimeAgo does not show 'just now'
+    // when adding a comment, doining the calculations below
+    // to find out if a comment was just posted. in that case
+    // removing the Twitter style
+    var fuzzy = null;
+    let timeDiff = moment().diff(time.toDate(), 'minutes');
+    if (timeDiff < 1) {
+      fuzzy = 'just now';
+    }
+
+    if (fuzzy === null) {
+      if (this.props.twitter) {
+        fuzzy = this.convertTime.format(time.toDate(), 'twitter');
+      } else {
+        fuzzy = this.convertTime.format(time.toDate(), this.options);
```

Comment:
`this.options` isn't used anymore so let's remove it. It could be confusing in the future for someone else looking at the code trying to understand what's going on.